### PR TITLE
Update webpack.config.js

### PR DESCRIPTION
**Pull request by brandonseydel**
Make the bundle dynamic and cache-busting for prod.

### CODE REVIEW

File: webpack/webpack.config.js [hunk 1]
```diff
@@ -57,11 +57,13 @@ module.exports = function(env, { /* @if jasmine || tape || mocha*/runTest, /* @e
...
     // @if !jasmine && !tape && !mocha
-    entry: './src/main./* @if babel */js/* @endif *//* @if typescript */ts/* @endif */',
+    entry: {
+      entry: './src/main./* @if babel */js/* @endif *//* @if typescript */ts/* @endif */'
+    },
     // @endif
     output: {
       path: path.resolve(__dirname, 'dist'),
-      filename: 'entry-bundle.js'
+      filename: production ? '[name].[contenthash].bundle.js' : '[name].bundle.js'
```
Comment (3cp) [inline webpack/webpack.config.js:66]: Probably need at least make test mode to use `entry-bundle.js`, currently the browser test (using my browser-do) asssumes the bundle name as `entry-bundle.js` regardless of bundler (webpack or dumber).

Probably needs to karma to replace the simple browser-do, as karma has better watch mode and coverage report  support.

Comment (brandonseydel) [reply webpack/webpack.config.js:66]: ok in non prod it still should be entry-bundle....hmmm

**Comment by 3cp** [comment]
Guess you missed some. This is from one of the error log.
```
sh: dist/entry-bundle.js: No such file or directory
npm ERR! code ELIFECYCLE
```

**Comment by 3cp** [comment]
You probably want to rebase/merge current master branch. There is a fix on latest webpack+html-webpack-plugin.

**Pull request merged**